functorch/csrc/VmapModeRegistrations.cpp

```diff
@@ -5,39 +5,15 @@
 // LICENSE file in the root directory of this source tree.
 
 #include <torch/library.h>
-#include <ATen/ATen.h>
-#include <functorch/csrc/VmapTransforms.h>
-#include <functorch/csrc/BatchedTensorImpl.h>
-#include <functorch/csrc/PlumbingHelper.h>
-#include <functorch/csrc/Constants.h>
-#include <functorch/csrc/DynamicLayer.h>
 
 
 namespace at {
 namespace functorch {
 
-void unsupportedRandomOp(const c10::OperatorHandle& op, torch::jit::Stack* stack) {
-  TORCH_CHECK(false, "vmap: We do not yet support calling random operations inside of vmap. ",
-              "Please perform random operations outside of vmap as a workaround");
-}
-
```

Comment:
(no action required) I can see some future where when someone adds a new randomness function to native_functions.yaml, we automatically generate an "unsupported" error message. But we can bring this back when that happens